src/Microsoft.Sbom.Constants/SpdxConstants.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.ObjectModel;
+using Microsoft.Sbom.Contracts;
+using Microsoft.Sbom.Contracts.Enums;
+using Microsoft.Sbom.Extensions.Entities;
+
+namespace Microsoft.Sbom.Constants;
+
+public static class SpdxConstants
+{
+    public const string ManifestFolder = "_manifest";
```

Comment:
`ManifestFolder` and `LoggerTemplate` are unrelated to SPDX. Can we put them back where they were in the Api project?

Reply:
We've discussed offline to revert all the changes related to constants in this PR since it requires a deeper look at interface restructuring.